packages/web/components/Player/Section/PlayerMemberships.tsx

```diff
@@ -90,17 +79,27 @@ const DaoListing: React.FC<DaoListingProps> = ({
   chain,
   address,
 }) => {
+  let explorerUrl = null;
+  if (chain) {
+    if (chain.toLowerCase() === 'xdai')
+      explorerUrl = `https://blockscout.com/xdai/mainnet/address/${address}`;
+    else if (chain.toLowerCase() === 'ethereum')
+      explorerUrl = `https://etherscan.io/address/${address}`;
+  }
+
   let message;
   if (memberXp !== undefined) {
     message = `XP: ${Math.floor(memberXp || 0)}`;
   } else {
     message = `Shares: ${memberShares}/${daoShares}`;
   }
 
-  const [isLink, setIsLink] = useState(false);
-
   return (
-    <LinkDao chain={chain || ''} address={address || ''} setIsLink={setIsLink}>
+    <LinkDao
```

Comment:
chain and address are no longer necessary for this component, maybe we can remove it?

Reply:
You're right, I'll remove them then we can merge this so you can rebase and merge yours